From 34b2a6a09a9558a16d4d3f63cce52579f7f557e7 Mon Sep 17 00:00:00 2001
From: karajas <karajas@microsoft.com>
Date: Wed, 18 Oct 2017 00:19:44 -0400
Subject: [PATCH 1/1] change packagetargetrid in sharedframework.proj

---
 src/sharedFramework/sharedFramework.proj | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/sharedFramework/sharedFramework.proj b/src/sharedFramework/sharedFramework.proj
index 94442cc..dd511d1 100644
--- a/src/sharedFramework/sharedFramework.proj
+++ b/src/sharedFramework/sharedFramework.proj
@@ -117,8 +117,8 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <TrimPkgsFromDeps Include="runtime.$(PackageTargetRid).microsoft.netcore.dotnetapphost" />
-      <TrimPkgsFromDeps Include="runtime.$(PackageTargetRid).microsoft.netcore.dotnethostresolver" />
+      <TrimPkgsFromDeps Include="runtime.$(TargetRid).microsoft.netcore.dotnetapphost" />
+      <TrimPkgsFromDeps Include="runtime.$(TargetRid).microsoft.netcore.dotnethostresolver" />
       <TrimPkgsFromDeps Include="microsoft.netcore.dotnetapphost" />
       <TrimPkgsFromDeps Include="microsoft.netcore.dotnethostresolver" />
     </ItemGroup>
-- 
1.8.3.1

