From 913ece232c9ee949f83a1fd43fdb6e1505d235e2 Mon Sep 17 00:00:00 2001
From: karajas <karajas@microsoft.com>
Date: Wed, 25 Oct 2017 18:51:54 -0400
Subject: [PATCH 1/1] Patch Microsoft.DotNet.Web.ItemTemplates version

---
 build/DependencyVersions.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/DependencyVersions.props b/build/DependencyVersions.props
index aa3ef16..f728197 100644
--- a/build/DependencyVersions.props
+++ b/build/DependencyVersions.props
@@ -32,7 +32,7 @@
    <MicroBuildVersion>0.2.0</MicroBuildVersion>
    <XliffTasksVersion>0.2.0-beta-000042</XliffTasksVersion>
 
-    <AspNetCoreTemplatePackageVersion>2.0.5</AspNetCoreTemplatePackageVersion>
+    <AspNetCoreTemplatePackageVersion>1.0.0-beta2-20170810-304</AspNetCoreTemplatePackageVersion>
     <!-- This should either be timestamped or notimestamp as appropriate -->
     <AspNetCoreRuntimePackageFlavor>notimestamp</AspNetCoreRuntimePackageFlavor>
     <!--BranchName and AspNetCoreVersion will not always be the same-->
-- 
1.8.3.1

