From 589760a9d40762f993b9da5f7478256fae328fc2 Mon Sep 17 00:00:00 2001
From: karajas <karajas@microsoft.com>
Date: Tue, 24 Oct 2017 13:14:04 -0400
Subject: [PATCH 1/1] unblock SPA app version

---
 build/BundledTemplates.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/BundledTemplates.props b/build/BundledTemplates.props
index a7a7069..cf4d207 100644
--- a/build/BundledTemplates.props
+++ b/build/BundledTemplates.props
@@ -6,6 +6,6 @@
 
     <BundledTemplate Include="Microsoft.DotNet.Web.ItemTemplates" Version="$(AspNetCoreTemplatePackageVersion)" />
     <BundledTemplate Include="Microsoft.DotNet.Web.ProjectTemplates.2.0" Version="$(AspNetCoreTemplatePackageVersion)" />
-    <BundledTemplate Include="Microsoft.DotNet.Web.Spa.ProjectTemplates" Version="$(AspNetCoreTemplatePackageVersion)" />
+    <BundledTemplate Include="Microsoft.DotNet.Web.Spa.ProjectTemplates" Version="1.0.417" />
   </ItemGroup>
 </Project>
-- 
1.8.3.1

